autokeras/task.py

```diff
@@ -333,3 +333,92 @@ def __init__(self,
             max_trials=max_trials,
             directory=directory,
             seed=seed)
+
+
+class TimeSeriesForecaster(auto_model.AutoModel):
+    """AutoKeras time series data forecast class.
+
+    # Arguments
+        column_names: A list of strings specifying the names of the columns. The
```

Comment:
Remind me (this is the same API as the structure data classes): what happens if the data is already vectorized (e.g. Numpy) and we don't specify column names or column types? What if you just have raw data without semantic annotations?

Reply:
1. The user does not need to specify the column_names and column_types if the data are already vectorized. Actually, we only need the column_names if (1) the data are raw data and (2) column_names is not in the data and (3) the user is using the column_names to specify the types of the data in the column_types.

2. The meta_model would infer the column semantic types by looking into the values of the data.